src/viking.js

```diff
@@ -1,11 +1,63 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health = health;
+        this.strength = strength;
+}
+    attack() {
+        return this.strength;
+}
+    receiveDamage(damage) {
+        this.health -= damage;
+    }
+}
+
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+    constructor(name, health, strength) {
+        super(health, strength);
+        this.name = name;
+        }
+        receiveDamage(damage){
+            super.receiveDamage(damage);
+            if (this.health > 0) {
+                return `${this.name} has received ${damage} points of damage`;
+            } return `${ this.name} has died in act of combat`;
+        } 
```

Comment:
In this code block, you can erase the `super.receiveDamage(damage);`. The reveiceDamage method should subtract the passed damage from the health property of the Viking object!